tests/block_log_util_test.py

```diff
@@ -71,7 +71,7 @@ def verifyBlockLog(expected_block_num, trimmedBlockLog):
     node0.kill(signal.SIGTERM)
 
     Print("Let's have node1's head advance a couple blocks")
```

Comment:
Update print statement (no longer a couple of blocks).

Reply:
https://github.com/AntelopeIO/spring/pull/1380/commits/a3171d85bd960e5ce6d5317e6348e662c5b75019